week-5-practise/index.html

```diff
@@ -1,157 +1,171 @@
-						<hr class="my-4">
-						<div class="buttons">
-							<a class="btn btn-primary btn-lrg" href="#">Donate a bike today</a>
-							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
-						</div>
+	<div id="content" class="container site-content">
+		<div class="alert alert-info" role="alert">
+			You've helped us give
+			<span id="donation-count-alert">72</span> bikes to refugees and asylum seekers.
+			<strong>Thank you!</strong>
+		</div>
+		<div class="row">
+			<div class="primary-content col-lg-8">
+				<div class="jumbotron">
+					<h1 class="display-3">Bikes for Refugees</h1>
+					<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in Scotland.</p>
+					<hr class="my-4">
+					<div class="buttons">
+						<a class="btn btn-primary btn-lrg" id="donate" href="#">Donate a bike today</a>
+						<a class="btn btn-secondary btn-lrg" id="volunteer" href="#">Volunteer</a>
 					</div>
-					<button id="blueBtn" type="button" class="btn btn-primary colorButton">Blue</button>
-					<button id="orangeBtn" type="button" class="btn btn-primary colorButton">Orange</button>
-					<button id="greenBtn" type="button" class="btn btn-primary colorButton">Green</button>
-					<br>
-					<h3>Register with us today:</h3>
-					<form >
-						<div class="form-group">
-							<label for="exampleInputEmail1">Email address</label>
-							<input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
-							<small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
-						</div>
-						<div class="form-group">
-							<label for="example-text-input">Your name</label>
-							<input class="form-control" type="text" value="" id="example-text-input">
+				</div>
+				<button id="blueBtn" type="button" class="btn btn-primary colorButton">Blue</button>
+				<button id="orangeBtn" type="button" class="btn btn-primary colorButton">Orange</button>
+				<button id="greenBtn" type="button" class="btn btn-primary colorButton">Green</button>
+				<br>
+				<h3>Register with us today:</h3>
+				<form>
+					<div class="form-group">
+						<label for="exampleInputEmail1">Email address</label>
+						<input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
+						<small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+					</div>
+					<div class="form-group">
+						<label for="example-text-input">Your name</label>
+						<input class="form-control" type="text" value="" id="example-text-input">
+					</div>
+					<div class="form-group">
+						<label for="exampleTextarea">Describe yourself</label>
+						<textarea class="form-control" id="exampleTextarea" rows="3"></textarea>
+					</div>
+					<button type="submit" class="btn btn-primary" onclick="ValidateRegisterForm()">Submit</button>
```

Comment:
It's mostly considered bad practice to use `onclick` DOM attributes now. It makes it more difficult to change the code and harder to read if the function calls are split between JS and HTML. Generally you should use `.addEventListener` instead.